src/components/TripCard/TripCard.js

```diff
@@ -2,23 +2,23 @@ import { useMutation } from '@apollo/client';
 import { CREATE_USER_TRIP_MUTATION } from '../../queries/queries';
 import './TripCard.css';
 
-// import { DELETE_TRIP_MUTATION } from '../../queries/queries';
+import { DELETE_TRIP_MUTATION } from '../../queries/queries';
 
 function TripCard({ trip, user }) {
-  const [createUserTrip, { loading, error, data }] = useMutation(CREATE_USER_TRIP_MUTATION);
-  // const [deleteTrip, { loading, error, data }] = useMutation(DELETE_TRIP_MUTATION);
-  console.log({loading})
-  console.log({error})
+  const [createUserTrip, { loading, error }] = useMutation(CREATE_USER_TRIP_MUTATION);
+  const [deleteTrip, { data }] = useMutation(DELETE_TRIP_MUTATION);
+  // console.log({loading})
+  // console.log({error})
   console.log({data})
+  console.log(trip.id)
```

Comment:
I want to mention that typically we don't leave `console.logs` in code, but I don't think its necessary to remove this for this project.

Reply:
Thanks! Yeah, we should have only been merging in clean code, but that'll definitely be a focal point for PRs going forward